### ISSUE TITLE
[PROJECT ONBOARDING] Aeraki Mesh

### ISSUE THREAD

**Issue by amye** [cncf/sandbox#174]
Welcome to CNCF Project Onboarding!
This is an issue created to help onboard your project into the CNCF after the TOC has voted to accept your project.
We would like to complete onboarding within one month of acceptance.

From the project side, please ensure that you:

- [x] Understand the project proposal process and reqs: <https://github.com/cncf/toc/blob/main/process/project_proposals.md#introduction>
- [x] Understand the services available for your project at CNCF <https://www.cncf.io/services-for-projects/>
- [x] Ensure your project meets the CNCF IP Policy: <https://github.com/cncf/foundation/blob/master/charter.md#11-ip-policy>
- [x] Review the online programs guidelines: <https://github.com/cncf/foundation/blob/master/online-programs-guidelines.md>
- [x] Understand the trademark guidelines: <https://www.linuxfoundation.org/en/trademark-usage/>
- [x] Understand the license allowlist: <https://github.com/cncf/foundation/blob/master/allowed-third-party-license-policy.md#approved-licenses-for-allowlist>
- [x] Is your project working on written, open governance? see <https://contribute.cncf.io/maintainers/governance/>
- [x] Slack: Are your slack channels migrated to the Kubernetes or CNCF Slack? (see <https://slack.com/help/articles/217872578-Import-data-from-one-Slack-workspace-to-another> for more details)
- [x] Is your project in its own separate neutral github organization?
- [x] Submitted a Pull request to add your project as a sandbox project to <https://landscape.cncf.io>
- [x] Create maintainer list + add to aggregated <https://maintainers.cncf.io> list by submitting a PR to it
- [x] Have added your project to <https://github.com/cncf/contribute>
- [x] Artwork: Submit a pull request to <https://github.com/cncf/artwork> with your artwork
- [x] Domain: transfer domain to the CNCF - <https://jira.linuxfoundation.org/plugins/servlet/theme/portal/2/create/63>

Things that CNCF will need from the project:

- [x] Provide emails for the maintainers added to <https://maintainers.cncf.io> in order to get access to the maintainers mailing list and ServiceDesk
- [x] Trademarks: transfer any trademark and logo mark assets over to the LF - <https://github.com/cncf/foundation/tree/master/agreements> has agreements
- [x] GitHub: ensure 'thelinuxfoundation' and 'caniszczyk' are added as initial org owners, this helps us make sure we have continuity of GH ownership
- [x] GitHub: ensure [DCO](https://github.com/apps/dco) or [CLA](https://github.com/cncf/cla) are enabled for all GitHub repositories of the project
- [x] GitHub: ensure that hat the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md) (or your adopted version of it) are explicitly referenced at the project's README on GitHub
- [x] Website: ensure LF footer is there and [website guidelines](https://github.com/cncf/foundation/blob/master/website-guidelines.md) followed (if your project doesn't have a dedicated website, please adopt those guidelines to the README file of your project on GitHub).
- [x] Website: Analytics transferred to projects@cncf.io
- [x] CII: Start on a CII best practices badge <https://bestpractices.coreinfrastructure.org/en>

Things that the CNCF will do or help the project to do:

- [x] Devstats: add to devstats <https://devstats.cncf.io/>
- [x] Insights: add to LFX Insights <https://insights.v3.lfx.linuxfoundation.org/>
- [x] Marketing: update relevant intro + slide decks
- [x] Events: update CFP + Registration + CFP Area forms
- [x] ServiceDesk: confirm maintainers have read <https://www.cncf.io/services-for-projects/>
- [x] CNCF Welcome Email Sent to confirm maintainer list access, welcome email has monthly project sync details
- [x] Create space for meetings/events on <https://community.cncf.io>, e.g., <https://community.cncf.io/pravega-community/> - (<https://github.com/cncf/communitygroups/blob/main/README.md#cncf-projects>)
- [x] Adopt a license scanning tool, like [FOSSA](https://fossa.com/) or [Snyk](https://snyk.io/)
- [x] Add project's groups.io maintainer list to [maintainers@cncf.io](https://groups.google.com/a/cncf.io/g/maintainers/members)

**Comment by krook** [comment]
Hi @Xunzhuo 

I'm working on closing out some of our older Sandbox onboarding checklists. It seems we have a few open tasks here to complete.

- [ ] Domain: transfer aeraki.net domain to the CNCF - https://jira.linuxfoundation.org/plugins/servlet/theme/portal/2/create/63
- [ ] Website: Google Analytics transferred to projects@cncf.io
- [ ] Adopt a license scanning tool, like FOSSA or Snyk @RobertKielty can help

**Comment by RobertKielty** [comment]
Hi @Xunzhuo, we have two license scanners configured to track compliance with the CNCF 3rd Party License Policy. I can send out invitations to the team to allow you to scan your code repos on either FOSSA or Snyk. 

Which service would you prefer to use?

**Comment by RobertKielty** [comment]
@zhaohuabing thank you for importing the main code repo into CNCF FOSSA.

There are 27 repos in https://github.com/aeraki-mesh org

@Xunzhuo and @krook plase review the repos in the org to see if any more repos need to be scanned to ensure compliance with the 3rd Party License policy.

The guidance would be that any repo that releases artefacts to the end-user community should be scanned for 3rd Party License Policy complilance. Not clear to me if any more repos should be added. Let me know.

**Comment by krook** [comment]
Hi @Xunzhuo and @zhaohuabing,

Just two more onboarding tasks left.

> - [ ] Transfer [your aeraki.net domain to the CNCF](https://jira.linuxfoundation.org/plugins/servlet/desk/portal/2/create/63?requestGroup=19). The "LF Stakeholder email" is [projects@cncf.io](mailto:projects@cncf.io). The "Project" is CNCF.
> - [ ] Website: Google Analytics transferred to [projects@cncf.io](mailto:projects@cncf.io). You can invite this address as an admin.

**Comment by zhaohuabing** [comment]
> Hi [@Xunzhuo](https://github.com/Xunzhuo), we have two license scanners configured to track compliance with the CNCF 3rd Party License Policy. I can send out invitations to the team to allow you to scan your code repos on either FOSSA or Snyk.
> 
> Which service would you prefer to use?

@RobertKielty these two repos need to be scanned, thanks!
https://github.com/aeraki-mesh/aeraki
https://github.com/aeraki-mesh/meta-protocol-proxy

**Comment by zhaohuabing** [comment]
> Hi [@Xunzhuo](https://github.com/Xunzhuo) and [@zhaohuabing](https://github.com/zhaohuabing),
> 
> Just two more onboarding tasks left.
> 
> > * [ ]  Transfer [your aeraki.net domain to the CNCF](https://jira.linuxfoundation.org/plugins/servlet/desk/portal/2/create/63?requestGroup=19). The "LF Stakeholder email" is [projects@cncf.io](mailto:projects@cncf.io). The "Project" is CNCF.[ ]  

https://github.com/cncf/sandbox/issues/174#issuecomment-2873610830

Website: Google Analytics transferred to [projects@cncf.io](mailto:projects@cncf.io). You can invite this address as an admin.

**Comment by krook** [comment]
Thanks @zhaohuabing, were there any analytics to transfer? It wasn't clear from your comment whether you had done that task or not.

**Comment by zhaohuabing** [comment]
> Thanks @zhaohuabing, were there any analytics to transfer? It wasn't clear from your comment whether you had done that task or not.

Hi @krook thanks for your help with the transferring. Somehow I can't open the Google analytics site.

**Comment by krook** [comment]
> Hi [@krook](https://github.com/krook) thanks for your help with the transferring. Somehow I can't open the Google analytics site.

Do you mean that you don't have access to it?

**Comment by krook** [comment]
@zhaohuabing thanks for initiating the domain transfer. Did you get access to Google Analytics? Are you able to invite [projects@cncf.io](mailto:projects@cncf.io) as an admin to it?

**Comment by zhaohuabing** [comment]
@krook Sorry I still got problems accessing the Google Analytics website. Is it possible that we change the [track id](https://github.com/aeraki-mesh/website/blob/55d0a9785e2764a1587a774fa76752b7876a298a/config.yaml#L14) to a CNCF one?

**Comment by zhaohuabing** [comment]
Unfortunately, Google analytics console was unstable - mostly likely became I accessed using a VPN.

I finally managed to get into the analytics console but found the existing ID is not there. Looks like this ID was not tracked.

**Comment by krook** [comment]
Ok, then it sounds like it's easiest just to create a new one since there'd be no history to worry about @chalin

**Comment by nate-double-u** [comment]
> @zhaohuabing @RobertKielty @nate-double-u: The website is hosted on Netlify, can one of you grant me access so that I can configure the new GA4 property?

I don't think we have access to their Netlify team, and I don't think their site is housed in a CNCF owned project team.
